src/processes.c

```diff
@@ -2083,20 +2087,24 @@ static int ps_read (void)
 			case 'T': stopped++;  break;
 			case 'W': paging++;   break;
 		}
+		if (ps_get_cmdline (
+		     pid, NULL, cmdline, sizeof(cmdline)) != NULL) {
+			cmdline_processes++;
+		}
 
-		ps_list_add (ps.name,
-				ps_get_cmdline (pid, ps.name, cmdline, sizeof (cmdline)),
-				&pse);
+		ps_list_add (ps.name, cmdline, &pse);
 	}
 
 	closedir (proc);
 
-	ps_submit_state ("running",  running);
-	ps_submit_state ("sleeping", sleeping);
-	ps_submit_state ("zombies",  zombies);
-	ps_submit_state ("stopped",  stopped);
-	ps_submit_state ("paging",   paging);
-	ps_submit_state ("blocked",  blocked);
+	ps_submit_state ("running",     running);
+	ps_submit_state ("sleeping",    sleeping);
+	ps_submit_state ("zombies",     zombies);
+	ps_submit_state ("stopped",     stopped);
+	ps_submit_state ("paging",      paging);
+	ps_submit_state ("blocked",     blocked);
+	ps_submit_state ("no_cmdline",  (total_processes - cmdline_processes));
```

Comment:
A single space after the `,` is enough. Then align the rest of the arguments with this.

Reply:
Done.